gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -34,205 +29,30 @@
-      return NotImplemented
-
-    return self < other or self == other
-
-  def __ne__(self, other):
-    return not self == other
-
-  def __gt__(self, other):
-    if not isinstance(other, Variant):
-      return NotImplemented
-
-    return other < self
-
-  def __ge__(self, other):
-    if not isinstance(other, Variant):
-      return NotImplemented
-
-    return other <= self
-
-
-class VariantCall(object):
-  """A class to store info about a variant call.
-
-  A call represents the determination of genotype with respect to a particular
-  variant. It may include associated information such as quality and phasing.
-  """
-
-  def __init__(self, name=None, genotype=None, phaseset=None, info=None):
-    # type: (str, List[int], str, Dict[str, Any]) -> None
-    """Initialize the :class:`VariantCall` object.
-
-    Args:
-      name: The name of the call.
-      genotype: The genotype of this variant call as specified by the VCF
-        schema. The values are either `0` representing the reference, or a
-        1-based index into alternate bases. Ordering is only important if
-        `phaseset` is present. If a genotype is not called (that is, a `.` is
-        present in the GT string), -1 is used.
-      phaseset: If this field is present, this variant call's genotype ordering
-        implies the phase of the bases and is consistent with any other variant
-        calls in the same reference sequence which have the same phaseset value.
-        If the genotype data was phased but no phase set was specified, this
-        field will be set to `*`.
-      info: A map of additional variant call information. The key is specified
-        in the VCF record and the type of the value is specified by the VCF
-        header FORMAT.
-    """
-    self.name = name
-    self.genotype = genotype or []
-    self.phaseset = phaseset
-    self.info = info or {}
-
-  def __eq__(self, other):
-    return ((self.name, self.genotype, self.phaseset, self.info) ==
-            (other.name, other.genotype, other.phaseset, other.info))
-
-  def __lt__(self, other):
-    if self.name != other.name:
-      return self.name < other.name
-    elif self.genotype != other.genotype:
-      return self.genotype < other.genotype
-    elif self.phaseset != other.phaseset:
-      return self.phaseset < other.phaseset
-    else:
-      return self.info < other.info
-
-  def __le__(self, other):
-    return self < other or self == other
-
-  def __gt__(self, other):
-    return other < self
-
-  def __ge__(self, other):
-    return other <= self
-
-  def __ne__(self, other):
-    return not self == other
-
-  def __repr__(self):
-    return ', '.join(
-        [str(s) for s in [self.name, self.genotype, self.phaseset, self.info]])
 
 
 class _ToVcfRecordCoder(coders.Coder):
   """Coder for encoding :class:`Variant` objects as VCF text lines."""
 
   def encode(self, variant):
-    # type: (Variant) -> str
+    # type: (vcf_parser.Variant) -> str
```

Comment:
Let's keep everything as 'Variant' now (it would be valid as it's still declared here). We'd need to change everything if we want to use the vcf_parser version not just this particular one.

Reply:
done.

Reply:
Looks like this was added back in the latest sync.